.github/CODEOWNERS

```diff
@@ -635,6 +635,7 @@ packages/kbn-reporting/mocks_server @elastic/appex-sharedux
 x-pack/plugins/reporting @elastic/appex-sharedux
 packages/kbn-reporting/public @elastic/appex-sharedux
 packages/kbn-reporting/server @elastic/appex-sharedux
+packages/kbn-reporting-share @elastic/appex-sharedux
```

Comment:
Maybe we could place the share code under `packages/kbn-reporting/public/share`, instead of creating a new package?